File: 344. Reverse String.php
class Solution {

/**
 * @param String[] $s
 * @return NULL
 */
function reverseString(&$s) {

    $i = 0;
    $j= count($s) -1;

    while($i < $j){
        $v = $s[$i];
        $s[$i] = $s[$j];
        $s[$j] = $v;

        $i++;$j--;
    }
    return $s;
}
}

File: Leet-code/344-reverse-string/344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
      
      int n     = s.size();
      int left  = 0;
      int right = n-1;
      int j     = 0;
      
      // using swap
      while(left < right){
      
        int temp = s[left];
        s[left]  = s[right];
        s[right] = temp;
        
        left++;
        right--;  
      }
      

        
    }
};